### DISCUSSION TITLE
Increasing tooltip font size

### DISCUSSION THREAD

**Discussion by Sahayavinodh** [jgraph/drawio#4412] [Q&A] [answered]
I want to increase the font size inside the tooltip. I want to know whether we have any options for it.

**Comment by alderg** [comment, marked as answer]
You can use HTML, eg. `<font size="3">This is a tooltip</font>`

**Reply by Sahayavinodh** [reply]
If I need to include html what is the plugin I need to include in mkdocs.yml
